test/Main.hs

```diff
@@ -37,5 +37,13 @@ tests :: ExchangeConf -> TestTree
 tests conf = testGroup "Tests"
         [ MarketData.tests conf
         , Private.tests    conf
-        , Socket.tests conf (ProductId "ETH-BTC") 
+        , Socket.tests     conf products Socket.allChannels
         ]
+
+-- TODO: Make this a list of all products
+products = [ ProductId "ETH-BTC"
+           , ProductId "ETH-EUR"
+           , ProductId "ETH-USD"
+           , ProductId "BTC-EUR"
+           , ProductId "BTC-USD"
+           ]
```

Comment:
I think we should update this to the latest list, it feels sloppy not to do so before we merge.

Reply:
I added this for testing purposes, to diversify the test data. Running the tests against every market might makes sense in some circumstances, but not as part of the automated test suite, I don't think. That said, it would be nice to have a list of all products and then maybe select a subset of markets to run the tests against as part of the automated suite, or let users choose what they want to test against.

Attempting to update the list just now, I ran `getProducts` and it returns only six products, despite there actually being dozens. I don't know what's going on there, but I don't know of another way to get a list of products. If this is a bug in Coinbase's code, we should make them aware of it, see #11. 

Also note that this is the sort of thing that will have to be kept up to date if it's meant to be complete, so we'll have to figure out how we want to do that.